account_template_active/README.rst

```diff
@@ -0,0 +1,127 @@
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--financial--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-financial-tools/tree/14.0/account_template_active
+    :alt: OCA/account-financial-tools
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_template_active
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/92/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to disable account template items, adding
+an ``active`` field on the following models:
+
+* ``account.account.template``
+
+.. figure:: https://raw.githubusercontent.com/OCA/account-financial-tools/14.0/account_template_active/static/description/account_account_template_tree.png
+
+* ``account.fiscal.position.template`` and it two related models
+    * ``account.fiscal.position.account.template``
+    * ``account.fiscal.position.tax.template``
+
+.. figure:: https://raw.githubusercontent.com/OCA/account-financial-tools/14.0/account_template_active/static/description/account_fiscal_position_template_form.png
+
+This module can be interesting in a multi company context,
+if you install a lot of accounting templates, with some changes.
+
+It can also be interesting if you want to developp a custom localization
+module based on the official one, but with some removed / added / altered items.
+
+**Note:**
+
+* The field ``active`` is present in the Odoo ``account`` module for the model
+  ``account.tax.template``. for that field, this module only adds the ``active``
+  field on the tree view.
+
+.. figure:: https://raw.githubusercontent.com/OCA/account-financial-tools/14.0/account_template_active/static/description/account_tax_template_tree.png
+
+For consistency reason,
+
+* if user disables an ``account.account.template`` it will disable the related
+  ``account.fiscal.position.account.template``
+
+* if user disables an ``account.tax.template`` it will disable the related
+  ``account.fiscal.position.tax.template``
+
+* if user enables an ``account.fiscal.position.account.template`` it will enable the
+  related ``account.account.template``
+
+* if user enables an ``account.fiscal.position.tax.template`` it will enable the
+  related ``account.tax.template``
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to Configuration > Templates
```

Comment:
- Go to Accounting -> Configuración -> Templates

Reply:
Done in readme/USAGE.rst. (the mentioned file is autogenerated)